packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects.
+        */
+        'func-names': ['error', 'as-needed'],
+        indent: ['error', 4],
+        'linebreak-style': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects or with a comment.
+            Including one class per file is a best practice and recommended by the Angular style
```

Comment:
nitpick/minor:

> ...is a best practice and is also the recommendation by the Angular style guide...

Basically to separate the two a bit: it is an Angular style guide recommendation, but it is also, beyond that, a best practice in general.